smpclient/transport/ble.py

```diff
@@ -80,12 +87,13 @@ def __init__(self) -> None:
         logger.debug(f"Initialized {self.__class__.__name__}")
 
     @override
-    async def connect(self, address: str, timeout_s: float) -> None:
-        logger.debug(f"Scanning for {address=}")
+    async def connect(self, timeout_s: float) -> None:
```

Comment:
This PR didn't change the behavior, but it got me wondering: What is the reasoning behind having a single parameter for both name and address? While this is probably fine for 99% of use-cases, you can easily make this connect to the wrong device by giving a device a name, that looks the same as the BLE address of another device :see_no_evil: :sweat_smile:

Reply:
Yeah, I don't love it, and thought about creating a union type that's name or address, but honestly, if someone uses a MAC address as the device name, I WANT them to open a ticket - force them out into the wild! 🤣